### PR TITLE
Use static from cdn. Fix styles

### PR DESCRIPTION
**Pull request by kosmos342**

### CODE REVIEW

File: yolapi/pypi/static/css/pygments.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/*
+Pygments 2.18.0
+Theme: default
+This css content is autogenerated by `pygmentize -S default -f html`
+*/
+
+pre { line-height: 125%; }
+td.linenos .normal { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
```
Comment (snitch) [inline yolapi/pypi/static/css/pygments.css:8]: Warning - Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

File: yolapi/pypi/static/css/pygments.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+span.linenos { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
```
Comment (snitch) [inline yolapi/pypi/static/css/pygments.css:9]: Warning - Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

File: yolapi/pypi/static/css/pygments.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+td.linenos .special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
```
Comment (snitch) [inline yolapi/pypi/static/css/pygments.css:10]: Warning - Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

File: yolapi/pypi/static/css/pygments.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+span.linenos.special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
```
Comment (snitch) [inline yolapi/pypi/static/css/pygments.css:11]: Warning - Adjoining classes: span.linenos.special (adjoining-classes)
Warning - Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

File: yolapi/pypi/static/css/pygments.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+.o { color: #666666 } /* Operator */
+.ch { color: #3D7B7B; font-style: italic } /* Comment.Hashbang */
+.cm { color: #3D7B7B; font-style: italic } /* Comment.Multiline */
+.cp { color: #9C6500 } /* Comment.Preproc */
+.cpf { color: #3D7B7B; font-style: italic } /* Comment.PreprocFile */
+.c1 { color: #3D7B7B; font-style: italic } /* Comment.Single */
+.cs { color: #3D7B7B; font-style: italic } /* Comment.Special */
+.gd { color: #A00000 } /* Generic.Deleted */
+.ge { font-style: italic } /* Generic.Emph */
+.ges { font-weight: bold; font-style: italic } /* Generic.EmphStrong */
```
Comment (snitch) [inline yolapi/pypi/static/css/pygments.css:25]: Warning - Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

File: yolapi/pypi/static/css/yolapi.css [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#packages td:nth-child(3) {
```
Comment (snitch) [inline yolapi/pypi/static/css/yolapi.css:1]: Warning - Don't use IDs in selectors. (ids)

File: yolapi/pypi/static/css/pygments.css (hunk 1 above)
Comment (kosmos342) [reply yolapi/pypi/static/css/pygments.css:8]: it's autogenerated

**Review by RayeN — APPROVED** [review]

**Pull request merged**